package.json

```diff
@@ -1,26 +1,22 @@
 {
   "name": "webrtc-stats-gatherer",
-  "version": "8.0.2",
+  "version": "9.0.0",
   "description": "Gathers stats on interval for webrtc peer connection",
-  "main": "dist/webrtc-stats-gatherer.cjs",
-  "browser": "dist/webrtc-stats-gatherer.min.js",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "files": [
     "src",
     "dist"
   ],
   "scripts": {
-    "lint": "semistandard",
-    "build": "npm run build-web && npm run build-node",
+    "clean": "rimraf dist lib",
+    "build": "npm run clean && tsc",
```

Comment:
Just to keep the scripts tidy, maybe `build-web` & `build-node` should be removed since we aren't using those anymore.